SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -100,6 +101,29 @@ VirtualWorkerId UGridBasedLBStrategy::WhoShouldHaveAuthority(const AActor& Actor
 	return SpatialConstants::INVALID_VIRTUAL_WORKER_ID;
 }
 
+SpatialGDK::QueryConstraint UGridBasedLBStrategy::GetWorkerInterestQueryConstraint() const
+{
+	// For a grid-based strategy, the interest area is the cell that the worker is authoritative over plus some border region.
+	check(IsReady());
+	check(InterestBorder >= 0);
```

Comment:
There could be a use case where this is desired idk? Seems quite aggressive to check here

Reply:
sure, can remove, it's gone